addons/website_sale/tests/test_website_sale_combo_configurator.py

```diff
@@ -0,0 +1,112 @@
+    def test_website_sale_combo_configurator(self):
+        no_variant_attribute = self.env['product.attribute'].create({
+            'name': "No variant attribute",
+            'create_variant': 'no_variant',
+            'value_ids': [
+                Command.create({'name': "A"}),
+                Command.create({'name': "B", 'is_custom': True, 'default_extra_price': 1}),
+            ],
+        })
+        product_a1 = self.env['product.template'].create({
+            'name': "Product A1",
+            'attribute_line_ids': [
+                Command.create({
+                    'attribute_id': no_variant_attribute.id,
+                    'value_ids': [Command.set(no_variant_attribute.value_ids.ids)],
+                }),
+            ],
+        })
+        combo_a = self.env['product.combo'].create({
+            'name': "Combo A",
+            'combo_item_ids': [
+                Command.create({'product_id': product_a1.product_variant_id.id, 'extra_price': 5}),
+                Command.create({'product_id': self._create_product(name="Product A2").id}),
+            ],
+        })
+        combo_b = self.env['product.combo'].create({
+            'name': "Combo B",
+            'combo_item_ids': [
+                Command.create({'product_id': self._create_product(name="Product B1").id}),
+                Command.create({'product_id': self._create_product(name="Product B2").id}),
+            ],
+        })
+        self._create_product(
+            name="Combo product",
+            list_price=25,
+            type='combo',
+            combo_ids=[
+                Command.link(combo_a.id),
+                Command.link(combo_b.id),
+            ],
+        )
+        self.start_tour('/', 'website_sale_combo_configurator', login='admin')
+
+    def test_website_sale_combo_configurator_single_configuration(self):
+        """ Test that the combo configurator isn't shown if there's a single configuration. """
+        no_variant_attribute = self.env['product.attribute'].create({
+            'name': "No variant attribute",
+            'create_variant': 'no_variant',
+            'value_ids': [Command.create({'name': "A"})],
+        })
+        product = self.env['product.template'].create({
+            'name': "Test product",
+            'attribute_line_ids': [
+                Command.create({
+                    'attribute_id': no_variant_attribute.id,
+                    'value_ids': [Command.set(no_variant_attribute.value_ids.ids)],
+                }),
+            ],
+        })
+        combo = self.env['product.combo'].create({
+            'name': "Test combo",
+            'combo_item_ids': [Command.create({'product_id': product.product_variant_id.id})],
+        })
+        self._create_product(
+            name="Combo product",
+            type='combo',
+            combo_ids=[Command.link(combo.id)],
+        )
+        self.start_tour('/', 'website_sale_combo_configurator_single_configuration', login='admin')
```

Comment:
```suggestion
        self.start_tour('/', 'website_sale_combo_configurator_single_configuration')
```
Probably can be tested as public user :eyes: